### PR TITLE
Build script changed

### LINKED ISSUE #1881: Update build scripts to support flavors
**Actual Behaviour**

Build script does not support flavors.

**Expected Behaviour**

playstore and fdroid favors need to be supported by the scripts when uploading to playstore and making releases in fdroid store

**Steps to reproduce it**

> Not available

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Issue for GSoC

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1881 

**Changes**: change build script 

**Screenshot/s for the changes**:N/A

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: WIP

### CODE REVIEW

File: scripts/upload-apk.sh [hunk 1]
```diff
@@ -13,28 +13,36 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
...
 
-\cp -r ../app/build/outputs/apk/*/**.apk .
-\cp -r ../app/build/outputs/apk/debug/output.json debug-output.json
-\cp -r ../app/build/outputs/apk/release/output.json release-output.json
+\cp -r ../app/build/outputs/apk/*/*/**.apk .
+\cp -r ../app/build/outputs/apk/fdroid/debug/output.json fdroid-debug-output.json
+\cp -r ../app/build/outputs/apk/playstore/debug/output.json playstore-debug-output.json
+\cp -r ../app/build/outputs/apk/fdroid/release/output.json fdroid-release-output.json
+\cp -r ../app/build/outputs/apk/playstore/release/output.json playstore-release-output.json
 \cp -r ../README.md .
```
Comment (adityastic) [inline scripts/upload-apk.sh]: @CloudyPadmal this should be removed, as in apk branch the readme would not be linked to the doc images. It's better to have a seperate readme for this branch. Copying the current readme should not be done. Let me know your thoughts on it

File: scripts/upload-apk.sh [hunk 2]
```diff
@@ -13,28 +13,36 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
 
 cd apk
 
-\cp -r ../app/build/outputs/apk/*/**.apk .
-\cp -r ../app/build/outputs/apk/debug/output.json debug-output.json
-\cp -r ../app/build/outputs/apk/release/output.json release-output.json
+\cp -r ../app/build/outputs/apk/*/*/**.apk .
+\cp -r ../app/build/outputs/apk/fdroid/debug/output.json fdroid-debug-output.json
```
Comment (adityastic) [inline scripts/upload-apk.sh]: Also the jsons can be removed @CloudyPadmal

Comment (CloudyPadmal) [reply scripts/upload-apk.sh]: Yes good catch. Only the APKs should be in the branch. @neel1998 can you update the PR with the change?

File: scripts/upload-apk.sh (hunk 1 above)
Comment (CloudyPadmal) [reply scripts/upload-apk.sh]: :+1: @neel1998 remove this copy for now. We'll create a README specific for APK branch later.

File: scripts/upload-apk.sh [hunk 3]
```diff
@@ -45,8 +34,10 @@ fi
 if [ "$TRAVIS_BRANCH" == "$DEVELOPMENT_BRANCH" ]; then
     echo "Push to development branch detected, generating apk..."
     # Rename apks with dev prefixes
-    mv app-debug.apk pslab-dev-debug.apk
-    mv app-release-unsigned.apk pslab-dev-unsigned.apk
+    mv app-playstore-debug.apk pslab-dev-debug.apk
+    mv app-fdroid-debug.apk pslab-dev-debug-fdroid.apk
+    rm app-playstore-release.apk
```
Comment (adityastic) [inline scripts/upload-apk.sh]: release-unsigned.apk

File: scripts/upload-apk.sh [hunk 4]
```diff
@@ -45,8 +34,10 @@ fi
...
+    rm app-fdroid-release.apk
```
Comment (adityastic) [inline scripts/upload-apk.sh]: same, release-unsigned.apk

**Comment by CloudyPadmal** [comment]
@adityastic can do a favor? Can you double check https://github.com/fossasia/open-event-organizer-android build gradle files with this and see if all the changes are done accordingly? Thanks a lot :+1:

File: .travis.yml [hunk 5]
```diff
@@ -24,10 +24,12 @@ cache:
 install:
     - echo yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-28"
 
+before_script:
+  - bash scripts/prep-key.sh
+
 script:
-  - 'if [ "$TRAVIS_BRANCH" != "apk" ]; then ./gradlew build --stacktrace; fi'
```
Comment (CloudyPadmal) [inline .travis.yml]: This was added to prevent unwanted build for pushes to apk branch. Revert

**Comment by adityastic** [comment]
> @adityastic can do a favor? Can you double check https://github.com/fossasia/open-event-organizer-android build gradle files with this and see if all the changes are done accordingly? Thanks a lot 👍

sure, let me recheck

File: .travis.yml (hunk 5 above)
Comment (adityastic) [reply .travis.yml]: @CloudyPadmal Its fine. there need not be a check for apk branch as apk branch does not has .tavis.yml file. There wont be a travis run for this branch which is correct.

**Comment by adityastic** [comment]
@CloudyPadmal These changes needs to done:
1. Apk branch should remove apks. It currently contains apks which are 7 months old.
2. Prep key should be changed to generate release apk for development branch as well.

Everything else looks PERFECT

**Comment by neel1998** [comment]
@adityastic thanks for the review. I will start working on the suggested changes.

**Comment by adityastic** [comment]
@CloudyPadmal now release another version of pslab, and your apk branch should get fixed after the branch is built

**Pull request merged**